src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/config/ResourceServerConfiguration.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.locationsinsideprison.config
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
+import uk.gov.justice.hmpps.kotlin.auth.dsl.ResourceServerConfigurationCustomizer
+
+@Configuration
+@Profile("train")
```

Comment:
don't you need to set this profile in the helm configuration for training environment?

Reply:
Yes I think I already do that

Reply:
I was expecting to see
```
    SPRING_PROFILES_ACTIVE: "logstash train"
```
in the helm configuration for training environment. Can't see it being set anywhere else - although you do have a train configuration profile but that is only for tests...